docs/README.md

```diff
@@ -481,49 +481,85 @@ Using session middleware will result in a sequence like this:
 Here is a simple example of how the built-in session middleware of Telegraf can be used to count photos.
 
 ```js
-const session = require('telegraf/session')
+const { session } = require('telegraf')
 
 const bot = new Telegraf(process.env.BOT_TOKEN)
 bot.use(session())
-// customize session make key algorithm
-//   bot.use(session((ctx) => `${ctx.from.id}:${ctx.chat.id}`))
-//
-// customize session storage backend
-//   bot.use(session(null, {
-//     getItem(name) { ... },
-//     setItem(name, value) { ... },
-//     deleteItem(name) { ... }
-//   }))
-bot.on('text', (ctx) => {
+bot.on('photo', (ctx) => {
   ctx.session.counter = ctx.session.counter || 0
```

Comment:
`ctx.session` is optional, but its contents aren't: https://github.com/telegraf/telegraf/blob/9478a32ee5e526a0a72f0c760ae50c6e44740186/src/session.ts#L24

```suggestion
  ctx.session ??= { counter: 0 }
```